src/redux/actions/jobseeker.ts

```diff
@@ -0,0 +1,28 @@
+import {
+  REGISTER_JOBSEEKER_DATA_REQUEST,
+  REGISTER_JOBSEEKER_DATA_SUCCESS,
+  REGISTER_JOBSEEKER_DATA_FAIL,
+  Credential,
+} from '../types'
```

Comment:
I've already made jobseeker register and login.

Reply:
Got it. Will change it to UPDATE_JOBSEEKER. Does it fit right?

Reply:
Yes, you can do the UPDATE_JOBSEEKER_REQUEST etc. Unfortunately, the registerJobseeker doesn't work yet; I think the backend team hasn't removed skills from the register yet.